firmware/projects/Demo/Interrupts/bindings.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "shared/periph/gpio.h"
+
+namespace bindings {
+
+    extern shared::periph::DigitalOutput& indicator;
+    extern void DelayMS(unsigned int ms);
+    extern void Initialize();
+    extern void AppLevelFunction();
+
+    }  // namespace bindings
```

Comment:
~~See comment about code formatting~~

Since you've already made unformatted commits, this code will not be fixed by that. You should run these commands from inside the Demo/Interrupts folder

pip install clang-format
clang-format *

That will fix all files. Commit them, and the pre commit action should succeed

Reply:
I ran these commands within the Demo/interrupts folder. Let me know if this fixed the files